plugins/com.google.cloud.tools.eclipse.appengine.localserver.test/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerDelegateTest.java

```diff
@@ -93,51 +98,36 @@ public void testCheckProjectFacets_appEngineStandardFacet() throws CoreException
   @Test
   public void testCheckConflictingId_defaultServiceIds() throws CoreException {
     delegate = getDelegateWithServer();
-    Function<IModule, String> alwaysDefault = new Function<IModule, String>() {
-      @Override
-      public String apply(IModule module) {
-        return "default";
-      }
-    };
+    // Functions.constant() should have inferred the return type too
```

Comment:
You're right that comment is wrong.  It should be:
```
// Functions.constant() should parameterize both the input and output types
```
This is what it should have been:
```
  private <InputType, OutputType> Function<InputType, OutputType> constant(final OutputType result) {
    return new Function<InputType, OutputType>() {
      @Override
      public OutputType apply(InputType input) {
        return result;
      }
    };
  }
```